collector/collector.go

```diff
@@ -426,58 +447,16 @@ func FetchData(ctx context.Context, rpc *goclient.Client, fs *firestore.Client)
 			}
 			totalBucket.VolumeUSD = totalBucket.VolumeUSD.Add(v.VolumeUSD)
 		}
+		for _, tb := range tokenBuckets0 {
+			tb.Reserve = pairLiquidity.Reserve0
```

Comment:
hmm, we're setting the reserve for a token to liquidity for a given pair? trying to sort this out, thought this was going to add the reserve amount for each pair to get a total for a token.

Reply:
oh, right, that should add.

Reply:
How about now?